server/protocols/webadmin/webadmin-jmap/pom.xml

```diff
@@ -52,6 +52,10 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>apache-james-mailbox-store</artifactId>
```

Comment:
I do need it, because of the `MailboxSessionMapperFactory` used to retrieve a message from only its `MessageId` in `MessageFastViewProjectionCorrector`